.github/workflows/add-version-to-db.yml

```diff
@@ -0,0 +1,11 @@
+name: release
+on:
+  release:
+    types: [published]
+jobs:
+  add_release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Add release version to db
+      run: |
+        curl -X POST "https://tasking-manager-tm4-production-api.hotosm.org/api/v2/system/release/"
```

Comment:
Could someone abuse this POST request? I can see below that the request will update the release version from GitHub itself, so it would probably be difficult to accidentally or otherwise insert incorrect information here. What are your thoughts?

Reply:
As you mentioned this endpoint internally fetches the version from GitHub it will be not be possible to insert incorrect information here. So i don't think people can actually abuse this request. 

We can make this endpoint restricted to sys admin role only, but for that we need to generate token with no expiry or longer expiry date as our session token expires every 7 days. Since people can't abuse this request i think we can leave it open to all users.

Reply:
OK, sounds good to me